assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -206,4 +206,88 @@ describe('ExecutionResults', () => {
     const svgText = screen.getByText('Select Checks now!');
     expect(svgText).toBeInTheDocument();
   });
+
+  it("should render ExecutionResults with successfully filtered 'passing' results", async () => {
+    const {
+      clusterID,
+      hostnames,
+      checks: [checkID1, checkID2],
+      loading,
+      catalog,
+      error,
+      executionLoading,
+      executionData,
+      executionError,
+    } = prepareStateData('passing');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        clusterName="test-cluster"
+        clusterScenario="hana_scale_up"
+        cloudProvider="azure"
+        hostnames={hostnames}
+        catalogLoading={loading}
+        catalog={catalog}
+        catalogError={error}
+        executionLoading={executionLoading}
+        executionData={executionData}
+        executionError={executionError}
+      />,
+      { route: `/clusters_new/${clusterID}/executions/last?health=passing` }
+    );
+
+    expect(screen.getByText('test-cluster')).toBeTruthy();
```

Comment:
I don't think we need all of these expects, they are already tested somewhere else.
I would only use the last 3 ones, which are the good ones for this scenario.
The same for the next test